README.md

```diff
@@ -26,15 +26,62 @@ It's built with [Hugo](https://gohugo.io/) and hosted at https://iceberg.apache.
 
 The Iceberg documentation site is actually constructed from two hugo sites. The first, is the landing page. The second site, 
 is the documentation site which contains the full Iceberg documentation, including the javadoc. The landing page and
-documentation sites are completely self-contained in the `./landing-page` and `./docs` directories, respectively.
+documentation sites are completely self-contained in the `./landing-page` and `./docs` directories, respectively. The Javadocs are in the `./javadoc` directory.
 
-# Landing Page Deployment
+# How to Contribute
+
+## Submitting Pull Requests
+
+All pull requests to update the latest documentations should be submitted to the [Iceberg repository](https://github.com/apache/iceberg).
+When Iceberg publishes a new release, the release manager copies all the documentation to this repository and cut a new version branch.
+
+Changes to the landing page should be submitted to this repository against the `main` branch.
```

Comment:
The landing-page site includes the common markdown files as well (releases, spec, community, etc). We should keep those in `apache/iceberg` too and have PRs opened there, right? That would be simpler--all docs PRs are opened in `apache/iceberg` unless you're hotfixing a previous version in which case you open the PR against that particular version's branch in `apache/iceberg-docs`.

Reply:
yes agree, forgot to update this sentence after updating the scripts